modules/cpmso-yaml-provider-spec-aws.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control-plane-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-provider-spec-aws_{context}"]
+= Sample AWS provider specification
+
+The provider specification must match that of the control plane machines created by the installation program. You can omit any field that is set in the failure domain section of the CR.
+// "that of the control plane machines" -> i.e., in their `machine` resource? We should explain how to reference this regardless.
+// "any field that is set in the failure domain section" = ? I don't see any overlap in the sample I have. Both have a placement.region, but this one is different than the ones for eh availability zones in the failure domain snippet.
+//In general, I'm not super sure what needs to be explained here, what a user can change, etc.
+
+.Sample AWS `providerSpec` values
+[source,yaml]
+----
+providerSpec:
+  value:
+    ami:
+      id: ami-<ami_id_string> <a>
+    apiVersion: machine.openshift.io/v1beta1
+    blockDevices:
+    - ebs: <b>
+        encrypted: true
+        iops: 0
+        kmsKey:
+          arn: ""
+        volumeSize: 120
+        volumeType: gp3
+    credentialsSecret:
+      name: aws-cloud-credentials <c>
+    deviceIndex: 0
+    iamInstanceProfile:
+      id: <cluster_id>-master-profile <d>
+    instanceType: m6i.xlarge <e>
+    kind: AWSMachineProviderConfig <f>
+    loadBalancers: <g>
+    - name: <cluster_id>-int
+      type: network
+    - name: <cluster_id>-ext
+      type: network
+    metadata:
+      creationTimestamp: null
+    metadataServiceOptions: {}
+    placement:
+      region: us-west-1 <h>
+    securityGroups:
+    - filters:
+      - name: tag:Name
+        values:
+        - <cluster_id>-master-sg <i>
+    subnet: {}
+    tags:
+    - name: kubernetes.io/cluster/<cluster_id> <j>
+      value: owned
+    - name: expirationDate
+      value: 2022-10-20T15:38+0000
```

Comment:
I would not include this, this is an openshift CI specific tag that we use to clean up resources once they've been created in the CI runs